### PR TITLE
Update CI files from main branch

### PR DESCRIPTION
**Pull request by pulpbot**
[noissue]

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = ./docs/*,*/migrations/*,./pulpcore/app/protobuf/*
```
Comment (mdellweg) [inline .flake8:2]: I think we must make the generated file lint.

File: pulpcore/app/protobuf/telemetry_pb2.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: telemetry.proto
+# flake8: noqa
```
Comment (mdellweg) [inline pulpcore/app/protobuf/telemetry_pb2.py:4]: Can we configure protobuf to emit this line on autogeneration?

File: .flake8 (hunk 1 above)
Comment (bmbouter) [reply .flake8:2]: How? I don't see how that's possible. Everytime it generates it's entirely overwritten.

The alternative I can think of is to have the code shipped as its own pypi package that would provide several benefits:
* deduplicate it across repos
* stop having to configure lint and black exceptions for it
* not have to vendor auto-generated code in either repo

File: pulpcore/app/protobuf/telemetry_pb2.py (hunk 2 above)
Comment (bmbouter) [reply pulpcore/app/protobuf/telemetry_pb2.py:4]: I don't think so, protobuf isn't a configurable tool for generation. If you can see a way though, let me know.

**Review by mdellweg — APPROVED** [review]

File: pulpcore/app/protobuf/telemetry_pb2.py (hunk 2 above)
Comment (mdellweg) [reply pulpcore/app/protobuf/telemetry_pb2.py]: Didn't find a way... So let's do differently.

File: template_config.yml [hunk 3]
```diff
@@ -31,6 +31,7 @@ disabled_redis_runners:
 docker_fixtures: true
 docs_test: true
 flake8: true
+flake8_ignore: ["./pulpcore/app/protobuf/*"]
```
Comment (fao89) [inline template_config.yml:34]: 👍🏽

**Review by fao89 — APPROVED** [review]

**Pull request merged**